### ISSUE TITLE
Does not work on iPad

### ISSUE THREAD

**Issue by myzinsky** [lasconic/ShareUtils-QML#1]
The ShareUtils did work like a charm for a while. It works on iOS and Android. However, on iPad it does not work anymore. Any ideas why?

**Comment by lasconic** [comment]
Wow, I had no idea someone was using this code. I wrote it 8 years ago... and I don't use it anymore. I can't help, sorry.

**Comment by myzinsky** [comment]
That is a pitty, so you know how that should be done today?

**Comment by lasconic** [comment]
Nope. A quick google search gives this https://www.qt.io/blog/2017/12/01/sharing-files-android-ios-qt-app
It refers to this repo, but apparently goes a bit further.
